library/flink-library/src/main/java/uk/gov/gchq/gaffer/flink/operation/handler/AddElementsFromFileHandler.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2017 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.gov.gchq.gaffer.flink.operation.handler;
+
+import org.apache.flink.streaming.api.environment.StreamExecutionEnvironment;
+import uk.gov.gchq.gaffer.operation.OperationException;
+import uk.gov.gchq.gaffer.operation.impl.add.AddElementsFromFile;
+import uk.gov.gchq.gaffer.store.Context;
+import uk.gov.gchq.gaffer.store.Store;
+import uk.gov.gchq.gaffer.store.operation.handler.OperationHandler;
+
+public class AddElementsFromFileHandler implements OperationHandler<AddElementsFromFile> {
+    @Override
+    public Object doOperation(final AddElementsFromFile op, final Context context, final Store store) throws OperationException {
+        final StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
```

Comment:
Just a thought - it might be better to use Flinks batch API to load from a file rather than the Streaming API.